src/core/agent/services/connectionService.ts

```diff
@@ -362,7 +362,7 @@ class ConnectionService extends AgentService {
     const signifyContacts = await this.props.signifyClient.contacts().list();
     const storageContacts = await this.connectionStorage.getAll();
     const unSyncedData = signifyContacts.filter(
-      (contact: KeriaContact) =>
```

Comment:
We should be able to remove the `KeriaContact` type completely, and import `Contact` everywhere